packages/next/package.json

```diff
@@ -75,6 +75,7 @@
     ]
   },
   "dependencies": {
+    "@capsizecss/metrics": "0.0.0-x-avg-char-width-20221107122037",
```

Comment:
Probably want to add that to `next/dist/compiled` instead with `taskfile.js`.

Reply:
Okay. Will look into that